src/main/harmonysolver.qml

```diff
@@ -134,27 +133,50 @@ MuseScore {
                             var splittedSymbols = component.split('-')
                             var firstSymbol = splittedSymbols[0]
                             var secondSymbol = splittedSymbols[1]
+//todo do osobnej funkcji
 
                             if (isAlterationSymbol(secondSymbol[0])) {
-                                symbols.push(new FiguredBass.BassSymbol(parseInt(secondSymbol[1]),
-                                                                        secondSymbol[0]))
+                                if (secondSymbol[0] === secondSymbol[1]){
+                                    if (parseInt(secondSymbol[2]) !== 8) {
```

Comment:
O co chodzi z grubsza z tym różne od 8?

Reply:
Żeby w symbolach basu cyfrowanego nie mieć 8, bo nie jest do niczego potrzebne.
Np 
9-8
7
To weźmiemy do symboli tylko 7